### ISSUE TITLE
Lerna bootstrap improvements

### ISSUE THREAD

**Issue by evertonfraga** [ethereumjs/ethereumjs-vm#680]
With the monorepo transition, we're now able to perform a `lerna bootstrap --ignore-scripts` to link all its packages together, as long as they are in a conformant version range. 

Some dependencies were not symlinked as expected, though, resulting in using to the npm packages (remote). Listed below:
- vm -> block ^2.2.2
- blockchain -> block ~2.2.2
  - block -> tx ^2.1.1

Additionally, I am working on improving how ts-node runs our packages' code, without having to use `npm run build` for each of our dependencies, for each change.

**Comment by evertonfraga** [comment]
```
$ lerna bootstrap --ignore-scripts
$ cd packages
$ ls -l */node_modules  | grep -E (node_modules|ethereumjs-blockchain|ethereumjs-block|ethereumjs-account|ethereumjs-common|ethereumjs-vm|ethereumjs-tx)

account/node_modules:

block/node_modules:
ethereumjs-blockchain -> ../../blockchain/src
ethereumjs-common -> ../../common/src
ethereumjs-tx -> ../../tx/src

blockchain/node_modules:
ethereumjs-block <========
ethereumjs-common -> ../../common/src
ethereumjs-tx <========

common/node_modules:

tx/node_modules:
ethereumjs-common -> ../../common/src

vm/node_modules:
ethereumjs-account -> ../../account/src
ethereumjs-block <========
ethereumjs-blockchain -> ../../blockchain/src
ethereumjs-common -> ../../common/src
ethereumjs-tx -> ../../tx/src
```

**Comment by holgerd77** [comment]
With the block library this is likely due to having done these 2.2.x releases on [separate branches](https://github.com/ethereumjs/ethereumjs-block/tree/release/v2.2.2) not merged back into `master` due to the 3.0.0 release [still outstanding](https://github.com/ethereumjs/ethereumjs-block/pull/74). This is something we should integrate into the next major version of the VM here, this is actually long-time outstanding.

**Comment by evertonfraga** [comment]
Right, so i need to merge these outstanding branches to the monorepo first!

thanks for highlighting.
